code/game/machinery/cloning.dm

```diff
@@ -486,8 +488,15 @@ GLOBAL_VAR_INIT(clones, 0)
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
 		to_chat(occupant, span_notice("<b>There is a bright flash!</b><br><i>You feel like a new being.</i>"))
-		to_chat(occupant, span_userdanger("You do not remember your death, how you died, or who killed you. <a href='https://forums.yogstation.net/help/rules/#rule-1_6'>See rule 1.6</a>.")) //yogs
-		log_combat(occupant, "was cloned with memory loss")
+		if(efficiency < 9) // T4 and below
+			to_chat(occupant, span_notice("You do not remember your death, how you died, or who killed you. <a href='https://forums.yogstation.net/help/rules/#rule-1_6'>See rule 1.6</a>.")) //yogs
+			log_combat(occupant, "was cloned with memory loss")
+		else if(efficiency == 9) // One T5 scanner
+			to_chat(occupant, span_notice("Miraculously, you remember <i>how</i> you died. You don't remember anything about anyone involved, though!"))
+			log_combat(occupant, "was cloned with memory loss except for the method and location")
+		else if(efficiency >= 10) // Two T5 scanners
+			to_chat(occupant, span_notice("Miraculously, you remember everything about your previous death!"))
+			log_combat(occupant, "was cloned with no memory loss")
```

Comment:
Do admins approve of this? This should be logged "Cloned (has memories of [x])"

Reply:
1. Kind of arbitrary, what exactly do you want, a signed petition? It was also decided in a recent council vote that the code determines whether you lose memory or not, see below for the rule text
![image](https://github.com/yogstation13/Yogstation/assets/28408322/09f4da9a-8f36-453c-8800-91ad9dded804)


2. Is logged via `log_combat`, see below for similar code from https://github.com/yogstation13/Yogstation/pull/19273 
![image](https://github.com/yogstation13/Yogstation/assets/28408322/6df55cb7-615e-456b-bfa2-5ab3fc919efa)